.drone.yml

```diff
@@ -1,13 +1,36 @@
+workspace:
+  base: /go
+  path: src/github.com/kowala-tech/kUSD
+
 pipeline:
 
-  test-and-build:
+  test:
+    image: golang:1.9
+    commands:
```

Comment:
Can all this be a Makefile task so it's reproducible locally?

Reply:
(by 'all this' I mean the commands that run the tests)

Reply:
yes, makes sense, didn't realise our pipeline was well setup